ros2trace/ros2trace/verb/__init__.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
I suppose this is a copy from somewhere else, but still a new file that you authored?

```suggestion
# Copyright 2023 Apex.AI, Inc.
```

Reply:
Yeah, I copied it from one of the `ros2cli` packages and only changed the name of the command (in single quotes) in the class docstring. I didn't change the copyright owner because of that, but I guess I can change it if you think I should.